<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>terrain</class>
 <widget class="QWidget" name="terrain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QTabWidget" name="onglets">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="ajouter">
      <attribute name="title">
       <string>ajouter</string>
      </attribute>
      <widget class="QPushButton" name="bouton_ajouter">
       <property name="geometry">
        <rect>
         <x>334</x>
         <y>212</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>80</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Terrain :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom_terrain">
       <property name="geometry">
        <rect>
         <x>232</x>
         <y>80</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>150</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>CapacitÃ© :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="capacite_terrain">
       <property name="geometry">
        <rect>
         <x>232</x>
         <y>140</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>230</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>annuler</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>96</x>
         <y>282</y>
         <width>631</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="rechercher">
      <attribute name="title">
       <string>rechercher</string>
      </attribute>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>116</x>
         <y>43</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Terrain :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="t_chercher">
       <property name="geometry">
        <rect>
         <x>192</x>
         <y>40</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="bouton_chercher">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>40</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>chercher</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView_2">
       <property name="geometry">
        <rect>
         <x>125</x>
         <y>121</y>
         <width>321</width>
         <height>201</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="supprimer">
      <attribute name="title">
       <string>supprimer</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Liste des terrains</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>90</y>
         <width>321</width>
         <height>271</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>390</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id :</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="id_supprimer">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>381</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="bouton_supprimer">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>380</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_rq">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>440</y>
         <width>551</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>capacite_terrain</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>nom_terrain</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
